### PR TITLE
XAMARIN-43636 : Linking all fields, properties and events to SfAvatarView

### PR DESCRIPTION
**Pull request by shivaniramakrishnan14**
Linking all fields, properties and events to SfAvatarView

### CODE REVIEW

**Comment by shivaniramakrishnan14** [comment]
rebuild

**Comment by shivaniramakrishnan14** [comment]
rebuild

File: Xamarin/Avatar-view/customization.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ control: AvatarView
 documentation: ug
 ---
 
-# Customization 
+# Customization in Xamarin Avatar View(SfAvatarView)
 
 ## Customization in Xamarin Avatar View(SfAvatarView)
```
Comment (Choza-rajan) [inline Xamarin/Avatar-view/customization.md:12]: remove this 2nd same text in the header

File: Xamarin/Avatar-view/how-to.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ control: AvatarView
 documentation: ug
 ---
 
-# How to
+# How to in Xamarin Avatar View(SfAvatarView)
```
Comment (Choza-rajan) [inline Xamarin/Avatar-view/how-to.md:10]: # How to set badge view to avatar

File: Xamarin/Avatar-view/how-to.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ control: AvatarView
...
 
 ## Set badge view to avatar
```
Comment (Choza-rajan) [inline Xamarin/Avatar-view/how-to.md:12]: remove this 2nd header

**Comment by shivaniramakrishnan14** [comment]
rebuild

**Comment by shivaniramakrishnan14** [comment]
rebuild

**Comment by Choza-rajan** [comment]
rebuild

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**